appsso-starter-java/README.md

```diff
@@ -2,6 +2,35 @@
 
 This repository provides an example application used to set up an authentication mechanism with AppSSO.
 
-## Usage
+## Getting Started
 
+1. Discover your `AppSSO` service offerings
+
+   ```bash
+   tanzu services classes list
+   ```
+
+   If there isn't one,
+
+   - ask your service operator `OR`
+   - [create your own `ClusterUnsafeTestLogin`](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-reference-api-clusterunsafetestlogin.html)
+
+2. Select the service offering you'd like to connect
+
+4. Claim an instance from the offering
+
+   ```bash
+   tanzu services class-claims create appsso-starter-java \
+     --class <your-selected-appsso-offering> \
+     --parameter workloadRef.name=appsso-starter-java \
+     --parameter redirectPaths='["/login/oauth2/code/appsso-starter-java"]'
+   ```
+
+5. Apply your workload
+
+   ```bash
+   kubectl apply -f config
+   ```
+
+For reference, see
 [Securing a Workload with AppSSO](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/app-sso-app-operators-secure-spring-boot-workload.html)
```

Comment:
should this point to 1.6 docs as well?

Reply:
Yes. Although our documentation will evolve up until the _Docs Freeze_. That includes the structure of our documentation and therefor we cannot guarantee that links won't break. 

How about we link to the AppSSO docs' landing page rather than linking deeply into it. This link will be stable:

```suggestion
[AppSSO's documentation](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-about.html)
```

Reply:
Yes, let's update it. I'm not sure if anything will change here directly, but it's safer to update.